editor/src/components/editor/actions/actions.ts

```diff
@@ -4171,6 +4173,29 @@ export const UPDATE_FNS = {
       return UPDATE_FNS.OPEN_EDITOR_TAB(openTab, updatedEditor)
     }
   },
+  UPDATE_CHILD_TEXT: (action: UpdateChildText, editor: EditorModel): EditorModel => {
```

Comment:
Looks to me like this will always remove any regular children on the selected element, rather than just replacing a text block child.

Reply:
That should be the case; the experiment was deliberately permissive. You could even enter things like `<h1>Hello World</h1>` there. Perhaps we need a better name for the action, eg `UPDATE_CHILD_CONTENT` ?

Reply:
Doing that would just result in that exact text appearing on the Canvas until the code file is then re-parsed, e.g.:
![Screenshot_20210107_144159](https://user-images.githubusercontent.com/1044774/103905444-86e0fa00-50f6-11eb-927a-9ef13c7fdd6b.png)

Reply:
Yep. We'll likely want to enhance this so that only hitting `Enter` actually confirms the entry (and then also causes re-parsing).

Reply:
The formula bar is limited now because it is not interactive for non-text children, with future improvements this updater function will change too.